src/app/routes/applications/governance-list/governance-list.component.ts

```diff
@@ -368,7 +370,27 @@ export class GovernanceListComponent implements OnInit, OnDestroy {
     this.dataSource.data = JSON.parse(JSON.stringify(this.origDatasource));
   }
 
+  createSubOrg(parentOrg: any) {
+    const dialogRef = this.dialog.open(NewOrganizationComponent, {
+      width: '600px',
+      data: {
+        viewType: ViewType.NEW,
+        parentOrg: JSON.parse(JSON.stringify(parentOrg)),
+        owner: parentOrg.owner
+      },
+      maxWidth: '100%',
+      disableClose: true
+    });
+
+    dialogRef.afterClosed()
+      .pipe(
+        filter(Boolean),
```

Comment:
I was reading a bit about `filter(Boolean)` and found this SO answer which suggests using a explicit filter for `undefined` so that the type is preserved. Have you seen this before? https://stackoverflow.com/a/57928681
```suggestion
        filter(v=> v!== undefined),
```

Reply:
Ah, maybe it's better to just provide the type like `filter<type>(Boolean)` but maybe that's not necessary in this case?

Reply:
hmm, interesting. Thanks for sharing this one.